packages/ui/src/machines/authenticator/index.ts

```diff
@@ -64,50 +69,92 @@ export function createAuthenticatorMachine() {
           ],
         },
         signIn: {
-          entry: 'spawnSignInActor',
-          exit: stopActor('signInActor'),
+          initial: 'spawnActor',
+          states: {
+            spawnActor: {
+              always: { actions: 'spawnSignInActor', target: 'runActor' },
+            },
+            runActor: {
+              entry: 'clearActorDoneData',
+              exit: stopActor('signInActor'),
+            },
```

Comment:
I split the spawn actor step into two parts:
1. `spawnActor` that consumes what's on the context and passes it to the new actor, and
2. `runActor` that removes any residue props that were already passed to the spawned actor, and runs the actor